scalyr_agent/log_processing.py

```diff
@@ -3287,12 +3291,19 @@ def find_matches(
                     checkpoint_state = None
                     # Get the last checkpoint state if it exists.
                     if matched_file in previous_state:
+                        log.info(
+                            "%s has previous state %s" % (matched_file, previous_state)
```

Comment:
This was the log message I was using in my test version, but when I was looking through the logs, I found it difficult to pick it out because it starts with the filename.  I'd change this to say something like:

`Copying %s from checkpoint %s`, then we can just search the logs for "Copying" and will match either of those lines.